cli/tc.go

```diff
@@ -5,10 +5,10 @@ import (
 )
 
 // wrap the common gh lib shared with my other tools. splits common GH code from this CLI tool's specific tooling code
-type tcServer struct {
+type TcServer struct {
 	tc.Server
 }
 
-func (f FlagData) NewServer() tcServer {
```

Comment:
cli/tc.go:12:31: unexported-return: exported method NewServer returns unexported type cli.tcServer, which can be annoying to use (revive)
```
func (f FlagData) NewServer() tcServer {
                              ^
```